ovirt/CentOS-7-x86_64-oVirt-r1.ks

```diff
@@ -0,0 +1,180 @@
+auth --enableshadow --passalgo=sha512
+reboot
+url --url="mirror.centos.org/centos/7/os/x86_64"
+firewall --enabled --service=ssh
+firstboot --disable
+ignoredisk --only-use=vda
+keyboard --vckeymap=us --xlayouts='us'
+# System language
+lang en_US.UTF-8
+repo --name "os" --baseurl="http://mirror.centos.org/centos/7/os/x86_64/" --cost=100
+repo --name "updates" --baseurl="http://mirror.centos.org/centos/7/updates/x86_64/" --cost=100
+repo --name "extras" --baseurl="http://mirror.centos.org/centos/7/extras/x86_64/" --cost=100
+
+# oVirt specific repos
+repo --name "centos-ovirt42" --baseurl="http://mirror.centos.org/centos/7/virt/x86_64/ovirt-4.2/" --cost=100
```

Comment:
are you sure you want to hardwire things to a specific version ? its going to get cumbersome having to maintain this

Reply:
We don't have a ovirt-latest repo, so unless we create a symlink to ovirt-4.2 and keep the symlink updated I don't see a way to maintain this. Also, it doesn't seem much different updating the ks file instead of a symlink on mirrors.centos.org.